specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -2873,6 +2918,44 @@
         "url": "http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/core/StopAnalyzer.html"
       }
     },
+    "CustomNormalizer": {
+      "discriminator": "@odata.type",
```

Comment:
There doesn't appear to be any polymorphism here, so why do you need a discriminator? Does our API even require `@odata.type` for normalizer definitions?

Reply:
I was following the pattern for `LexicalAnalyzer`. If it's not needed, I can remove. I've also updated with the `LexicalNormalizer` base class for `CustomNormalizer`

Reply:
you're right, I don't think we need it right now. I'll remove.

Reply:
We do need `@odata.type` for normalizer definitions. We only support `#Microsoft.Azure.Search.CustomNormalizer` right now.